flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/batch/sql/TableScanTest.scala

```diff
@@ -210,10 +210,13 @@ class TableScanTest extends TableTestBase {
 
   @Test
   def testTableApiScanWithTemporaryTable(): Unit = {
-    util.tableEnv.connect(new FileSystem().path(tempFolder.newFile.getPath))
-        .withFormat(new OldCsv())
-        .withSchema(new Schema().field("word", DataTypes.STRING()))
-        .createTemporaryTable("t1")
+    util.tableEnv.createTemporaryTable("t1", TableDescriptor.forConnector("datagen")
```

Comment:
(The actual filesystem connector isn't available here, but we don't really need it anyway)